app/assets/stylesheets/organisms/_payment_form.scss

```diff
@@ -0,0 +1,176 @@
+.order-total {
+    font-weight: bold;
+}
+.tar {
+    text-align: right;
+}
+/* Credit Card Form */
+#js-creditcard-form {
+    font-size: 13px;
+    color: white !important;
+    position: relative;
+    padding: 0px;
+    width: 300px;
+    margin-right: auto;
+}
+#js-creditcard-form .wrapper {
+    border: 1px solid #CCC;
+    border-top: 1px solid #AAA;
+    border-right: 1px solid #AAA;
+    height: 90px;
+    width: 300px;
+    position: relative;
+    -webkit-border-radius: 5px;
+    -moz-border-radius: 5px;
+    border-radius: 5px;
+}
+#js-creditcard-form .input-group {
+    position: absolute;
+    top: 300px;
+}
+#js-creditcard-form .input-group.nmb_a {
+    position: absolute;
+    width: 200px;
+    top: 0px;
+    left: 0px;
+}
+#js-creditcard-form .input-group.nmb_b {
+    position: absolute;
+    width: 100px;
+    top: 0px;
+    right: 0px;
+}
+#js-creditcard-form .input-group.nmb_b input, #js-creditcard-form .input-group.nmb_d input {
+    text-align: center;
+}
+#js-creditcard-form .input-group.nmb_c {
+    position: absolute;
+    width: 200px;
+    top: 44px;
+    left: 0px;
+}
+#js-creditcard-form .input-group.nmb_d {
+    position: absolute;
+    width: 100px;
+    top: 44px;
+    right: 0px;
+}
+#js-creditcard-form input {
+  color: white !important;
+    background: none;
+    display: block;
+    width: 100%;
+    padding: 10px;
+    -moz-box-sizing: border-box;
+    -webkit-box-sizing: border-box;
+    box-sizing: border-box;
+    margin:0px;
+    padding-left: 35px;
+    border: none;
+}
+#js-creditcard-form .input-group .icon {
+    position: absolute;
+    width: 22px;
+    height: 22px;
+    background: #CCC;
+    left: 8px;
+    top: 7px;
+}
+#js-creditcard-form .input-group.nmb_a input {
+    border-right: 1px solid #ECECEC;
+}
+#js-creditcard-form .input-group.nmb_c input {
+    border-top: 1px solid #ECECEC;
+    border-right: 1px solid #ECECEC;
+}
+#js-creditcard-form input::-webkit-input-placeholder {
+    font-size: 12px;
+    text-transform: none;
+}
+#js-creditcard-form .input-group.nmb_d input {
+    border-top: 1px solid #ECECEC;
+}
+#js-creditcard-form .input-group.nmb_c input {
+    text-transform: uppercase;
+}
+#js-creditcard-form .accept {
+    color: #999;
+    font-size: 11px;
+    margin-bottom: 5px;
+}
+#js-creditcard-form .footer {
+    margin-top: 3px;
+    position: relative;
+    margin-left: 5px;
+    margin-right: 5px;
+}
+#js-creditcard-form .footer img {
+    padding: 0px;
+    margin: 0px;
+}
+#js-creditcard-form .iugu-btn {
+    position: absolute;
+    top: 0px;
+    right: 0px;
+}
+/* Do not forget to store your images in a secure server */
+#js-creditcard-form .input-group .icon.ccic-name {
+    background: url("http://storage.pupui.com.br/9CA0F40E971643D1B7C8DE46BBC18396/assets/ccic-name.1cafa1882fdd56f8425de54a5a5bbd1e.png") no-repeat;
```

Comment:
This SCSS is from Iugu. Probably this will change when @cbgabe or @filipelinhares put some more design.